src/parser/chiquito.lalrpop

```diff
@@ -136,21 +136,52 @@ ParseEq: Expression<BigInt, Identifier> = {
     <lhs: ParseArith> "==" <rhs: ParseArith> => build_bin_op("==", lhs, rhs),
     <lhs: ParseArith> "!=" <rhs: ParseArith> => build_bin_op("!=", lhs, rhs),
 
-    ParseArith
+    <lhs: ParseArith> "<" <rhs: ParseArith> => build_bin_op("<", lhs, rhs),
+    <lhs: ParseArith> "<=" <rhs: ParseArith> => build_bin_op("<=", lhs, rhs),
+    <lhs: ParseArith> ">" <rhs: ParseArith> => build_bin_op(">", lhs, rhs),
+    <lhs: ParseArith> ">=" <rhs: ParseArith> => build_bin_op(">=", lhs, rhs),
+
+    ParseBitOr
 }
 
+ParseBitOr: Expression<BigInt, Identifier> =  ParseLogichBinOp<"|", ParseBitXor>;
+
+ParseBitXor: Expression<BigInt, Identifier> =  ParseLogichBinOp<"^", ParseBitAnd>;
+
+ParseBitAnd: Expression<BigInt, Identifier> =  ParseLogichBinOp<"&", ParseArith>;
+
 // Arithmetical expressions
 ParseArith: Expression<BigInt, Identifier> = {
-    ParseSum
+    ParseLeftShift
 }
 
-ParseSum: Expression<BigInt, Identifier> = ParseArithBinOp<"+", ParseMul>;
+ParseLeftShift: Expression<BigInt, Identifier> = ParseArithBinOp<"<<", ParseRightShift>;
```

Comment:
@rutefig << and >> have the same presence so you need to group them like in ParseEq